cred/src/main/webapp/WEB-INF/web.xml

```diff
@@ -150,14 +154,20 @@
       <param-name>ca.nrc.cadc.vosi.AvailabilityPlugin</param-name>
       <param-value>org.opencadc.cred.ServiceAvailability</param-value>
     </init-param>
-    <load-on-startup>3</load-on-startup>
+    <load-on-startup>2</load-on-startup>
```

Comment:
this really should be 3 in case it (ever) relies on the init action having completed